libs/brig-types/src/Brig/Types/Swagger.hs

```diff
@@ -440,8 +446,12 @@ teamInvitation = defineModel "TeamInvitation" $ do
   property "created_by" bytes' $ do
     description "ID of the inviting user"
     optional
-  property "name" string' $
```

Comment:
Why did you remove the `name` field? there is both an `invitee_name` and a `name` field in the  invitation type

Reply:
ah so `name` refers to `inviter` name?  and hte old docs were just wrong?

Reply:
> Why did you remove the name field? there is both an invitee_name and a name field in the invitation type
> ah so name refers to inviter name? and hte old docs were just wrong?

Actually `name` was just wrong, it was never there. We had only `inviter_name` which was the inviter's name to be sent on the email so I just removed it.

However, and to keep it "consistent", I will use only `name`: as in, all the data there refers to the invitee, unless state otherwise. So I _actually_ added the `name` field instead and continue using `phone`. https://github.com/wireapp/wire-server/commit/cd11363ebe07bcb5517fa073a2c1ff127e291181